src/lib/crypto.cpp

```diff
@@ -158,6 +159,7 @@ pgp_generate_seckey(const rnp_keygen_crypto_params_t *crypto,
         goto end;
         break;
     }
+after_seckey_alg:
```

Comment:
See the previous - it would be enough to add comment instead of code modifications.